lib/hcrypto/rsa-ltm.c

```diff
@@ -536,7 +537,7 @@ ltm_rsa_generate_key(RSA *rsa, int bits, BIGNUM *e, BN_GENCB *cb)
     do {
 	BN_GENCB_call(cb, 2, counter++);
 	CHECK(random_num(&p, bitsp), 0);
-	CHECK(mp_find_prime(&p,128), MP_YES);
+	CHECK(mp_prime_next_prime(&p, trials, 0), MP_OKAY);
```

Comment:
This should be `CHECK(mp_prime_next_prime(&p, mp_prime_rabin_miller_trials(bitsp), 0), MP_OKAY);`

Reply:
@lhoward has `int trials = mp_prime_rabin_miller_trials(bitsp);` above, probably so as to avoid having to wrap this line.  It's just a style thing.